crates/turborepo-ui/src/lib.rs

```diff
@@ -39,9 +50,8 @@ impl UI {
         Self { should_strip_ansi }
     }
 
-    // STUB
-    pub fn is_ci(&self) -> bool {
-        env::var("CI").is_ok()
+    pub fn is_ci() -> bool {
+        turborepo_ci::is_ci() && std::io::stdout().is_terminal()
```

Comment:
I'm pretty sure we want this to be true when we don't have a tty
```suggestion
        turborepo_ci::is_ci() && !std::io::stdout().is_terminal()
```